pages/player_page/[id].md

```diff
@@ -33,8 +33,8 @@
 
 <LastRefreshed/>
 
-# <Value data={basic_info} column=name /> 
-<img src={basic_info[0].logo} class="h-16" />
+# <center> <Value data={basic_info} column=name /> </center>
```

Comment:
Can we maybe use the big value thing here to do the image on the left and the name to the right of it?

Reply:
big value no do images